desktop/core/src/desktop/js/apps/editor/components/sqlScratchpad/SqlScratchpad.vue

```diff
@@ -116,27 +135,42 @@
         minLines: null
       };
 
-      const initializeExecutor = async (): Promise<void> => {
+      const initialize = async (): Promise<void> => {
+        if (apiUrl.value) {
+          setBaseUrl(apiUrl.value);
+        }
+
+        if (user.value !== null && pass.value !== null) {
+          try {
+            await login(user.value, pass.value, jwt.value);
```

Comment:
And there depending on the auth we grab sessionid/csrf or JWT?

Then could use a js store lib?
(keep a is_authenticated flag, user names, profile infos, auth method used...)

Reply:
It should intercept tokens/cookies etc. automatically, logic is there but not fully tested.